cloudbuild-preprod.yaml

```diff
@@ -27,6 +27,15 @@ steps:
     args: [ 'run', 'deploy', 'sds', '--image', 'europe-west2-docker.pkg.dev/ons-sds-dns/sds/sds-preprod:latest',
             '--region', 'europe-west2', '--allow-unauthenticated', '--ingress', 'internal-and-cloud-load-balancing' ]
 
+  - name: 'gcr.io/cloud-builders/gcloud'
+    id: 'Show image vulnerabilities'
+    entrypoint: sh
+    args:
+      - '-c'
+      - |
+        gcloud alpha container images describe europe-west2-docker.pkg.dev/${_PROJECT_ID}/sds/sds-preprod:$SHORT_SHA \
```

Comment:
The image file for prod should be europe-west2-docker.pkg.dev/ons-sds-dns/sds/sds-preprod:latest.
Can this step be moved before "Run image" and add similar condition as in cloudbuild.yaml to block image deployment if severe vulnerabilities are found?

Reply:
Yes, it would be ideal to pause the image run step if vulnerability is found but the
image scanning doesn't produce result till after the image is run. So the step has to go after the run. 
There might be a workaround this but will need more investigation and better suited for a separate ticket with new PR.